### PR TITLE
Feature/vta-629 - EU Vehicle Category Page

### PR DESCRIPTION
**Pull request by BrandonT95**

### CODE REVIEW

**Comment by BrandonT95** [comment]
simulator_screenshot_4422E167-E1D3-4846-9C23-9CA1BFB7988B

**Comment by BrandonT95** [comment]
Screenshot 2022-06-01 at 12 01 23

**Review by FBarrassDVSA — APPROVED** [review]

File: src/app/pages/testing/test-creation/eu-vehicle-category/eu-vehicle-category.module.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+@NgModule({
+  declarations: [CategoryReadingPage],
+  imports: [
+    IonicModule,
+    CommonModule
+  ],
+  providers: [VehicleService]
+})
+export class RegionReadingPageModule {}
```
Comment (tomcrawleyy) [inline src/app/pages/testing/test-creation/eu-vehicle-category/eu-vehicle-category.module.ts:18]: this module name seems wrong

File: src/app/pages/testing/test-creation/eu-vehicle-category/eu-vehicle-category-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    path: '',
+    component: RegionReadingPageModule,
+  },
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule],
+})
+export class RegionReadingPageRoutingModule {}
```
Comment (tomcrawleyy) [inline src/app/pages/testing/test-creation/eu-vehicle-category/eu-vehicle-category-routing.module.ts:19]: this module name seems wrong too

**Review by tomcrawleyy — APPROVED** [review]

**Pull request merged**